Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -1,13 +1,24 @@
 #include "BatteryStateOfChargeService.h"
+#include <QDebug>
+#include <QFile>
+#include <QString>
+#include <QTextStream>
 
 namespace
 {
     const double BATTERY_AMP_HOUR_CAPACITY = 123.0;
+    const double MS_TO_HOURS = 3600000.0;
+    const double S_TO_HOURS = 3600.0;
+    const double HOUR_TO_M = 60.0;
 }
 
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    initalAmpHour_=BATTERY_AMP_HOUR_CAPACITY-BATTERY_AMP_HOUR_CAPACITY*initialStateOfChargePercent_/100;
```

Comment:
Use initialization lists for constructing variables - see `initialStateOfCharPercent_`
````
: x_(0)
, y_(1)
, etc..
````